.gitignore

```diff
@@ -115,6 +115,7 @@ tools/*.dll
 # Sensitive files
 *.keys
 *.pfx
+sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurations.xml
```

Comment:
As something to clean-up, in general we don't like to generate files into the source tree during the build. This file should either be committed or if it needs to be generated then it should get generated under the intermediate directory.